<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XfireEditAccount</class>
 <widget class="QWidget" name="XfireEditAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Account preferences - Xfire</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabPreferences">
     <property name="minimumSize">
      <size>
       <width>460</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabBasic">
      <attribute name="title">
       <string>&amp;Basic</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="gpbAccount">
         <property name="title">
          <string>Account information</string>
         </property>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="hblAccount">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lblUsername">
              <property name="text">
               <string>&amp;Xfire username:</string>
              </property>
              <property name="buddy">
               <cstring>prefUsername</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="prefUsername">
              <property name="toolTip">
               <string>The account name of your Yahoo account.</string>
              </property>
              <property name="whatsThis">
               <string>The account name of your Yahoo account.  This should be in the form of an alphanumeric string (no spaces).</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Kopete::UI::PasswordWidget" name="prefPassword" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="prefExcludeConnect">
            <property name="whatsThis">
             <string>If you check this checkbox, the account will not be connected when you press the &quot;Connect All&quot; button, or at startup when automatic connection at startup is enabled.</string>
            </property>
            <property name="text">
             <string>Exclude from connect all</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpbRegistration">
         <property name="title">
          <string>Registration</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="lblRegistration">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>If you do not currently have a Xfire account, please click the connect the button to create one in order to connect.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hblRegistration">
            <item>
             <spacer name="hspRegistrationLeft">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbtRegister">
              <property name="text">
               <string>Register new account</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hspRegistrationRight">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="gpbGeneral">
         <property name="title">
          <string>General preferences</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="prefGameDetection">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Use game detection</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="prefInformAccounts">
            <property name="text">
             <string>Inform other accounts about ingame status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="prefFriendsOfFriends">
            <property name="text">
             <string>Show friends of friends</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vspGeneral">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabConnection">
      <attribute name="title">
       <string>&amp;Connection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="gpbConnection">
         <property name="title">
          <string>Connection preferences</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="prefOverrideServer">
            <property name="text">
             <string>Override default server information</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wdgOverrideServer" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="hspOverrideServer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="lblServerPort">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Server / port:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KLineEdit" name="prefServer">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Only modify these values if you want to use a special IM proxy server, like SIMP</string>
               </property>
               <property name="whatsThis">
                <string>Only modify these values if you want to use a special IM proxy server, like SIMP</string>
               </property>
               <property name="text">
                <string>cs.xfire.com</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="prefPort">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Only modify these values if you want to use a special IM proxy server, like SIMP</string>
               </property>
               <property name="whatsThis">
                <string>Only modify these values if you want to use a special IM proxy server, like SIMP</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>65535</number>
               </property>
               <property name="value">
                <number>25999</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gpbVersion">
         <property name="title">
          <string>Xfire version</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblVersion">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="prefVersion">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="prefUpdateVersion">
            <property name="text">
             <string>Update version automatically</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Kopete::UI::PasswordWidget</class>
   <extends></extends>
   <header>kopetepasswordwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabPreferences</tabstop>
  <tabstop>prefUsername</tabstop>
  <tabstop>prefExcludeConnect</tabstop>
  <tabstop>pbtRegister</tabstop>
  <tabstop>prefGameDetection</tabstop>
  <tabstop>prefInformAccounts</tabstop>
  <tabstop>prefOverrideServer</tabstop>
  <tabstop>prefServer</tabstop>
  <tabstop>prefPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
